Meta/gn/README.md

```diff
@@ -6,7 +6,32 @@ GN is a metabuild system. It always creates ninja files, but it can create some
 
 This is a good [overview of GN](https://docs.google.com/presentation/d/15Zwb53JcncHfEwHpnG_PoIbbzQ3GQi_cpujYwbpcbZo/edit#slide=id.g119d702868_0_12).
 
-For more information, motivation, philosophy, and inspriation, see the LLVM documentation on its [GN build](https://github.com/llvm/llvm-project/tree/main/llvm/utils/gn#quick-start)
+For more information, motivation, philosophy, and inspiration, see the LLVM documentation on its [GN build](https://github.com/llvm/llvm-project/tree/main/llvm/utils/gn#quick-start)
+
+# Creating a gn build
+
+To create a GN build, you need to have GN installed. You can install it via homebrew on macOS, or via your package manager on Linux.
+On Ubuntu 22.04, the main package repos do not have an up to date enough package for GN, so you will need to get a binary from Google, or build it from source.
+The instructions are [here](https://gn.googlesource.com/gn/+/refs/heads/main#getting-a-binary).
+
+Once you have GN installed, you can create a build directory by running the following commands:
+
+```sh
+gn gen out
+```
+
+``gn gen`` creates a ninja build in the ``out`` directory. You can then build the project with ninja:
+
+```sh
+ninja -C out
+```
+
+If GN or ninja report a bunch of errors, it's likely that you need to create an ``args.gn`` that points to all the required tools.
+``args.gn`` belongs at the root of the build directory, and can be placed there before running gn gen, or modified with
+``gn args <build dir>``. See the section below for a typical ``args.gn``.
+
+After modifying ``args.gn``, be sure to run ``gn gen`` again to regenerate the ninja files.
```

Comment:
FYI shouldn't be needed if modifying via `gn` itself - only needed when editing manually in e.g. vim

Reply:
Also FYI all these double backticks could just be single backticks for inline code